src/main/java/com/denizenscript/denizencore/utilities/ReflectionHelper.java

```diff
@@ -93,15 +93,57 @@ public Field getFirstOfType(Class fieldClazz) {
             return null;
         }
 
-        public Field get(Object name) {
-            Field f = getNoCheck(name.toString());
+        public MethodHandle getGetter(String name) {
+            return toHandleGetter(get(name));
+        }
+
+        public MethodHandle getGetter(String name, Class<?> expected) {
+            return toHandleGetter(get(name, expected));
+        }
+
+        private MethodHandle toHandleGetter(Field f) {
+            if (f == null) {
+                return null;
+            }
+            try {
+                return LOOKUP.unreflectGetter(f);
+            }
+            catch (IllegalAccessException e) {
+                echoError(e);
+                return null;
+            }
+        }
+
+        public MethodHandle getSetter(String name) {
+            return toHandleSetter(get(name));
+        }
+
+        public MethodHandle getSetter(String name, Class<?> expected) {
+            return toHandleSetter(get(name, expected));
+        }
+
+        private MethodHandle toHandleSetter(Field f) {

```

Comment:
these two should probably be `public static`?

Reply:
Done